apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationActionsBar.jsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react'
 import django from 'django'
+import { HoverButton } from './HoverButton'
 
 export const ModerationNotificationActionsBar = (props) => {
   const translated = {
-    blockText: django.pgettext('kosmo', ' Block'),
-    unblockText: django.pgettext('kosmo', ' Unblock'),
-    replyText: django.pgettext('kosmo', ' Add statement'),
-    unarchiveText: django.pgettext('kosmo', ' Unarchive'),
+    blockText: django.pgettext('kosmo', 'Block'),
+    unblockText: django.pgettext('kosmo', 'Unblock'),
+    isBlockedText: django.pgettext('kosmo', 'Blocked'),
+    replyText: django.pgettext('kosmo', 'Add statement'),
```

Comment:
this is now too close to icon

Reply:
check it, looked wrong on my browser

Reply:
okay i will triple check, for me there is a space. Anyway should we include spacing in a translation key? that could lead to redundancy no? I think we should separate text from spacing

Reply:
okay you were right, the space didn't appear. Using a span now, hope that works